cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -609,3 +609,39 @@ func ifErrRemove(pErr *error, path string) {
 		}
 	}
 }
+
+func GetClusterAPIObject(client Client, clusterName, namespace string) (*clusterv1.Cluster, *clusterv1.Machine, []*clusterv1.Machine, error) {
```

Comment:
minor nit: needs export comment.
do we have a linter in place that catches these?